runtime/compiler/env/j9method.cpp

```diff
@@ -8409,7 +8409,7 @@ TR_J9ByteCodeIlGenerator::runFEMacro(TR::SymbolReference *symRef)
 
          uintptr_t methodHandle;
          uintptr_t methodDescriptorRef;
-         intptr_t methodDescriptorLength;
+         uintptr_t methodDescriptorLength;
```

Comment:
This change, and the similar changes below and in `handleServerMessage()` seem unnecessary

If we're concerned about possible overflow when adding 1 for the NUL terminator to size a buffer, then `uintptr_t` can avoid that, but on 32-bit `intptr_t` will be the same as `int32_t`, so overflow would still be possible. However, I don't think we're in any danger of passing a string remotely long enough to cause that overflow. It might be better to just adjust the assertion in `getStringUTF8Length()` to make sure that the result is always strictly less than the max, which would guarantee that we can always add 1 without overflow

Reply:
I'm not sure why I had changed this declaration - I'll revert it to `intptr_t`.

Regarding the change below, from `int` to `intptr_t`, that one was intentional - all of the other declarations of `methodDescriptorLength` in this file declare it as `intptr_t`.  I didn't see a reason why that one should be different, so I changed it for consistency.  Similarly for the declarations as type `size_t` and `int` in `handleServerMessage()`.